signup.js

```diff
@@ -0,0 +1,112 @@
+if (localStorage.getItem("accessToken")) {
+    location.replace("/folder");
+}
+
+const eInput = document.getElementById('email');
+const pInput = document.getElementById('password');
+const pDupInput = document.getElementById('password-dup');
+
+const emailErrMsg = document.getElementById('email-err-msg');
+const pwErrMsg = document.getElementById('password-err-msg');
+const pwDupMsg = document.getElementById('password-dup-msg');
+const signupbtn = document.querySelector('#signupbtn');
+
+const eChecker = /^[A-Za-z0-9_\.\-]+@[A-Za-z0-9\-]+\.[A-za-z0-9\-]+/; 
+    //@을 기준으로 앞 구간이 알파벳 or 숫자 조합으로 이루어져 있는지 체크, 뒷 구간이 알파벳 or 숫자 조합으로 이루어져 있는지 체크, 뒷 구간에서 . 뒷 구간이 알파벳 or 숫자 조합으로 이루어져 있는지 체크
+const pChecker = /^(?=.*[A-Za-z])(?=.*\d)[A-Za-z\d]{8,}$/;
+
+
+
+function emailCheck(email) {
+    if(eChecker.test(email) === false) {
+        return false; 
+    } else { 
+        return true; 
+    }
```

Comment:
[중요도: 높음] 불필요한 if문 같네요.
```suggestion
    return eChecker.test(email)
```
위처럼 처리하면 eChecker.test(email)가 true일 경우 true로 false이면 false로 알아서 처리해줄수 있겠죠?